e2e/test/BulkOperationsE2ETests.cs

```diff
@@ -12,12 +12,12 @@ namespace Microsoft.Azure.Devices.E2ETests
     [TestClass]
     [TestCategory("E2E")]
     [TestCategory("IoTHub")]
-    public class BulkOperationsE2ETests : E2ETestBase, IDisposable
+    public class BulkOperationsE2ETests : E2EMsTestBase, IDisposable
     {
         private readonly string DevicePrefix = $"E2E_{nameof(BulkOperationsE2ETests)}_";
         private readonly string ModulePrefix = $"E2E_{nameof(BulkOperationsE2ETests)}_";
 
-        [TestMethod]
+        [MsTestMethod]
```

Comment:
Why rename all these attributes? That is going to have massive impact across the source code.